tasks/managed/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -130,72 +136,152 @@ spec:
+            issue_id=$(jq -r '.fbc.issueId // empty' "${DATA_FILE}")
+            if [ -z "${issue_id}" ]; then
+              echo "Hotfix releases requires the issue id set in the 'fbc.issueId' key of the ReleasePlanAdmission " \
+                   "spec.data field"
+              exit 1
+            fi
+            target_index="${target_index}-${issue_id}-${timestamp}"
+          elif [ "${pre_ga}" = "true" ]; then
+            if [ -z "${product_name}" ] || [ -z "${product_version}" ]; then
+              echo "Pre-GA releases require 'fbc.productName' and 'fbc.productVersion' set in the " \
+                   "ReleasePlanAdmission spec.data field"
+              exit 1
+            fi
+            target_index="${target_index}-${product_name}-${product_version}-${timestamp}"
+          fi
+
+          echo "Processing fragment: ${fbc_fragment}:"
+          # The internal-request script will create the InternalRequest and wait until it finishes to get its status
+          # If it fails (Failed, Rejected or Timed out) the script will exit and display the reason.
+          echo "Creating InternalRequest to add FBC contribution to index image:"
+          internal-request --pipeline "update-fbc-catalog" \
+              -p fromIndex="$(params.fromIndex)" \
+              -p targetIndex="${target_index}" \
+              -p fbcFragment="${fbc_fragment}" \
+              -p iibServiceAccountSecret="${iib_service_account_secret}" \
+              -p publishingCredentials="${publishing_credentials}" \
+              -p buildTimeoutSeconds="${build_timeout_seconds}" \
+              -p buildTags="${build_tags}" \
+              -p addArches="${add_arches}" \
+              -p hotfix="${hotfix}" \
+              -p stagedIndex="${staged_index}" \
+              -p taskGitUrl="$(params.taskGitUrl)" \
+              -p taskGitRevision="$(params.taskGitRevision)" \
+              --service-account "${internal_request_service_account}" \
+              -l ${pipelinerun_label}="$(params.pipelineRunUid)" \
+              -t "${request_timeout_seconds}" |tee "$(workspaces.data.path)"/ir-"$(context.taskRun.uid)"-output.log
+
+          internalRequest=$(awk -F"'" '/created/ { print $2 }' \
+            "$(workspaces.data.path)"/ir-"$(context.taskRun.uid)"-output.log)
+          echo "done (${internalRequest})"
+
+          # Fetching InternalRequest status and populating results
+          results=$(kubectl get internalrequest "${internalRequest}" -o jsonpath='{.status.results}')
+          echo "${results}" > "$(workspaces.data.path)/ir-$(context.taskRun.uid)-result.json"
+
+          completion_time="$(jq -r '.jsonBuildInfo | fromjson | .updated' <<< "${results}")"
 
-        mustPublishIndexImage=$(jq -r '.genericResult | fromjson' <<< "${results}" | jq -r '.publish_index_image')
-        fbc_opt_in=$(jq -r '.genericResult | fromjson' <<< "${results}" | jq -r '.fbc_opt_in')
+          # construct the results json
+          build_results=$(jq \
```

Comment:
Could you add `index_image_resolved` to build results? It's needed by https://github.com/konflux-ci/release-service-catalog/pull/981, and seems https://github.com/konflux-ci/release-service-catalog/pull/984 intends to use it as well.

Reply:
https://github.com/konflux-ci/release-service-catalog/pull/984 FYI